### ISSUE TITLE
Starting Steam in Big Picture Mode - PS3 Dualshock Controller has wrong bindings  

### ISSUE THREAD

**Issue by fallchildren** [ValveSoftware/steam-for-linux#4015]
If I start Steam directly in Big Picture Mode my PS3 Controller has wrong bindings. Since the newest beta I even cannot change them. Strange thing is if I start Steam normally and then switch to BPM everything is fine and the controller bindings are set up correctly. It makes no different if I set the option "start Steam in Big Pictore Mode" or use the command line flag.

**Comment by kisak-valve** [comment]
Hello @fallchildren, can you please retest this issue with the 2018-05-18 steam beta client?

**Comment by kisak-valve** [comment]
Closing pending feedback.

**Comment by dubigrasu** [comment]
Sorry, I only now noticed this bug report, and I can confirm that is still happening with both stable or beta client. It used to work previously (although I can't be sure when exactly) and it was properly recognized as a PlayStation 3 controller with no extra configuration needed, basically working out of the box.
Now it is seen as a Generic X-input controller and no matter what settings I use (or if I enable Generic Gamepad Configuration Support) it is unusable. The left stick and left bumper are working OK but the face buttons ( X, Triangle, Square etc) are not.

**Comment by kisak-valve** [comment]
Thanks for the retest @dubigrasu.

**Comment by fallchildren** [comment]
Sorry for the very long delay but I did not have time to look into steam for some time now. Just retested it with current stable and beta and for me it works perfectly now. 

TL;DR
By the way cool stuff and good work on the "new" (no idea for how long you got that now) SteamOS/PS4/xinput controller configuration and mapping stuff. I like the idea very much that you're not vendor locking  people and are offering the same functionality on the controller level.

Just got a question: is there a place to put game specific issues with the new controller configuration system (the witcher 2 beta for linux is broken for me)?

**Comment by kisak-valve** [comment]
Hello @fallchildren, thanks for retesting. In general, individual game devs should be contacted with issues with their game and we'll track general controller issues with Valve titles in this issue tracker.

@dubigrasu, has your issue cleared up as well?

**Comment by kisak-valve** [comment]
Closing as the original issue is resolved.
